static/src/javascripts/projects/common/modules/commercial/ticker.js

```diff
@@ -16,7 +17,7 @@ const percentageTotalAsNegative = (end: number) => {
 
 const animateBar = (parentElement: HTMLElement) => {
     // If we've exceeded the goal then extend the bar 20% beyond the total
-    const end = total > goal ? total + total * 0.2 : goal;
+    const end = total > goal && allowToExceed ? total + total * 0.2 : goal;
```

Comment:
some brackets would help me parse this

Reply:
so the linter removes extra brackets, and setting `'no-extra-parens': 'off'` in the eslintrc files makes no difference. This is really annoying but I've wasted enough time on it for now